.pre-commit-config.yaml

```diff
@@ -0,0 +1,56 @@
+repos:
```

Comment:
Most of the checks here are related to Odoo addons.
Can you remove or fix their configuration if you want to keep them ?

Reply:
In fact there is 2 pull requests for this change, you can update an already existing one.
I proposed a configuration to use here: https://github.com/camptocamp/odoo-dump-bag/pull/16#pullrequestreview-430419816

Reply:
@p-tombez You are right, thanks. thus we removed the pre-commit-config.yaml.

Reply:
The configuration I proposed on https://github.com/camptocamp/odoo-dump-bag/pull/16#pullrequestreview-430419816
was fine though.